packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -32,6 +40,37 @@ export class GuxAvatar {
   @Prop()
   accent: GuxAvatarAccent = 'default';
 
+  /**
+   * Shows presence such as away or available.
+   * Must be combined with badge or ring props to take effect.
+   */
+  @Prop()
+  presence: GuxAvatarPresence = 'available';
```

Comment:
This all looks good to me now except presence defaulting to "available". I think we need a "none" option that is the default.